Master onpremise docker

christopherBinaural opened the pull request.

Commit:
Modificando cosas para onpremise docker sin swarm servicios separados.

christopherBinaural committed:
Eliminando auto detect addons limitado a carpetas especificas ahora todo lo que se encuentre en src se agregara a los addons.

Actualice el nodejs

Modifique el dockerfile y otros archivos para que variables del env referentes al odoo conf sean tomadas en cuenta

Commit:
Modifivcando compose de base de datos para soportar configs

Commit:
Merge branch 'master_onpremise_docker_db' of github.com:binaural-dev/docker-odoo into master_onpremise_docker

Commit:
Modificaciones para tener servicio de base de datos y odoo en yml diferentes.

Agregue configuracion de accesos de postgresql  y odoo.conf de nginx para la compatibilidad con workers.

miguel-binaural commented on .env_example:
```diff
@@ -28,33 +48,61 @@ DOMAIN={catchall:.*}
+#####################################
 TRAEFIK_HOST_PORT=
 TRAEFIK_SERVICE_CONTAINER_PORT=8080
+
```
Quitar traefik, ya usas nginx @christopherBinaural

Commit:
Correcciones.

Agregue pg_hba.example

corregui comentario en script odoo

agregue pg_hba original al .gitignore

elimine menciones al traefik en .env_example
